make/utils_exposed.cmake

```diff
@@ -193,6 +193,9 @@ function (DynamoRIO_get_full_path out target loc_suffix)
         set(output_dir ${runtime_dir})
       endif ()
     endif ()
+    # XXX i#3278: DR's win loader can't handle a path like that until full support has
+    # been implemented in convert_to_NT_file_path.
```

Comment:
Please add a comment to convert_to_NT_file_path as well so that someone hitting a problem in the future will see it and not have to spend time re-figuring this out.

Reply:
Done